odoo/addons/base/models/ir_actions.py

```diff
@@ -17,8 +17,30 @@
 from pytz import timezone
 
 _logger = logging.getLogger(__name__)
+_server_action_logger = _logger.getChild("server_action_safe_eval")
 
 
+class LoggerProxy:
```

Comment:
Can you add a small docstring explaining that we don't want to expose too much of logging.logger as it is used in safe_eval?